compose/ui/ui/src/desktopTest/kotlin/androidx/compose/ui/ComposeSceneInputTest.kt

```diff
@@ -228,17 +228,19 @@ class ComposeSceneInputTest {
 
         scene.sendPointerEvent(PointerEventType.Release, Offset(20f, 20f))
         background.events.assertReceivedNoEvents()
-        cutPopup.events.assertReceivedLast(
+        cutPopup.events.assertReceived(
             PointerEventType.Release, Offset(20f, 20f) - cutPopup.origin)
-        overlappedPopup.events.assertReceivedNoEvents()
+        cutPopup.events.assertReceivedLast(
```

Comment:
We probably have a bug here. We shouldn't send Enter in the line 222 to `cutPopup` in the first place, because (20, 20) is inside `overlappedPopup`. By the current design, Enter/Exit should be send independently of the current `gestureOwner`.

Reply:
> By the current design, Enter/Exit should be send independently of the current `gestureOwner`.

This PR does not change current sending Enter/Exit events during Move events. It just adds additional check/send round during Release event handling.

> We shouldn't send Enter in the line 222 to `cutPopup` in the first place

Maybe, but it's different thing. Sending `Exit` to `cutPopup` and `Enter` to `overlappedPopup` seems right here